src/interactive/Dropdown.jsx

```diff
@@ -10,18 +10,9 @@ import {
 
 import bindAll from "../utils/bindAll";
 
-const ConditionalWrap = ({condition, wrap, children}) => condition ? wrap(children) : children;
+export const DROPDOWN_MENU_ITEM_CLASS = 'dropdownMenu-item';
 
-export const Item = ({isActive, isSelected, children}) => (
-	<div
-		className="dropdownMenu-item"
-		style={{
-			backgroundColor: isActive && C_COOLGRAYLIGHTTRANSP
-		}}
-	>
-		{children}
-	</div>
-);
+const ConditionalWrap = ({condition, wrap, children}) => condition ? wrap(children) : children;
 
 /**
  * @module Dropdown
```

Comment:
Do we need to change the propType for `items` to expect an array of elements?

Reply:
Aha, yes.